### DISCUSSION TITLE
The app does not open

### DISCUSSION THREAD

**Discussion by user4302** [mifi/lossless-cut#783] [Support] [answered]
idk when this started, but the app no longer opens, i even downloaded the latest release EXE, and added it to the whitelist on my antivirus and ran as admin,
nothing works.
i feel like this started after i updated  FFmpeg, but im not sure.

edit: it seems my virus guard(360 TS) blocks it no matter what, unless i disable the antivirus temporarily

**Comment by mifi** [comment, marked as answer]
I assume you download llc from github. You could try an older version. if that doesn't work, then I don't know, because nothing has changed with the app recently.

**Reply by user4302** [reply]
yeah i got it from github i believe,
its my virusguard, its been blocking multiple apps, 
so LLC is all good.
